kgforge/specializations/stores/nexus/service.py

```diff
@@ -265,6 +266,9 @@ def resolve_context(self, iri: str, local_only: Optional[bool] = False) -> Dict:
             else:
                 raise ValueError(f"{context_to_resolve} is not resolvable")
         else:
+            # Make sure context is not deprecated
+            if resource['_deprecated']:
```

Comment:
I would check _deprecated is in the dict first.

Reply:
is there a case where it is not?